tests/auxiliary/minicore.rs

```diff
@@ -61,7 +62,8 @@ pub auto trait Unpin {}
 
 impl_marker_trait!(
     Copy => [
-        bool, char,
+        // `char` deliberately omitted for `coherence-impls-copy` test
+        bool,
```

Comment:
Question: could you slightly elaborate why `char` is specifically omitted? Looking at `tests/ui/coherence/coherence-impls-copy.rs`, AFAICT it doesn't even use `minicore`? If anything, that test uses the *real* (host, I think) std?

Reply:
I remember needing to add this but can't work out why it was necessary now - removing it.